resources/assets/js/angular/app.js

```diff
@@ -86,11 +86,11 @@ angular.module('fc', [
       }) 
       .state('root.admin.privacy', {
          url: 'privacy',
-         templateUrl: 'angular/admin/privacy.tpl.html'
+         templateUrl: 'templates/admin/privacy.tpl.html'
       })
       .state('root.admin.contact', {
          url: 'contact',
-         templateUrl: 'angular/admin/contact.tpl.html'
+         templateUrl: 'templates/admin/contact.tpl.html'
       });
 })
 .run(['$rootScope', '$state', 'userService', function ($rootScope, $state, userService) {
```

Comment:
If globstar works, all the better.  If not, I'm ok too.  Most tutorials actually suggest separating things into folders like js/, html/, and css/, so doing it that way would be a bit more "standardized" I guess.  I just like it my way :)